app/dts/bindings/zmk,underglow-indicators.yaml

```diff
@@ -6,30 +6,30 @@ description: Underglow indicators
 compatible: "zmk,underglow-indicators"
 
 properties:
-  bat-lhs:
+  bat-self:
     type: array
-    required: true
-  bat-rhs:
+    default: []
+  bat-other:
     type: array
-    required: true
+    default: []
   capslock:
     type: int
-    required: true
+    default: -1
```

Comment:
Not sure what the right way to define "nothing" was here, since I don't think these are nullable. 🤷